src/components/OpenLayersMap.js

```diff
@@ -185,6 +196,8 @@ class OpenLayersMap extends React.Component{
       })
     });
 
+
+
```

Comment:
Any reason for the newlines?

Reply:
Slipped and banged my head on the spacebar

Reply:
Goooone.